Scripts/DCS-BIOS/lib/modules/aircraft_modules/F-16C_50.lua

```diff
@@ -0,0 +1,2016 @@
+--DEST_OA1
+DEDLayout_l5["DEST OA1 DEST_OA1_ELEV"] = { 2, 3 }
+DEDLayout_l5["DEST OA1 ELEV"] = { 8, 8, 0, "_inv", "I" }
+DEDLayout_l5["DEST OA1 Asterisks_ELEV_both"] = { 7, 1, 16, "", "I" }
+DEDLayout_l5["DEST OA1 P3lbl"] = { 21, 3 }
+--DEST_OA2
+DEDLayout_l5["DEST OA2 DEST_OA2_BRG"] = { 2, 3 }
+DEDLayout_l5["DEST OA2 ELEV"] = { 8, 8, 0, "_inv", "I" }
+DEDLayout_l5["DEST OA2 Asterisks_ELEV_both"] = { 7, 1, 16, "", "I" }
+DEDLayout_l5["DEST OA2 P4lbl"] = { 21, 3 }
+--CRUS-TOS
+DEDLayout_l5["TOS_REQ_GS_label"] = { 4, 7 }
+DEDLayout_l5["TOS_REQ_GS_value"] = { 13, 7 }
+--CRUS-EDR
+DEDLayout_l5["EDR_WIND_AT_ALTITUDE"] = { 7, 4 }
+DEDLayout_l5["EDR_WIND_AT_ALTITUDE_Value"] = { 13, 10 }
+--CRUS RNG
+DEDLayout_l5["RNG_WIND_AT_ALTITUDE"] = { 7, 4 }
+DEDLayout_l5["RNG_WIND_AT_ALTITUDE_Value"] = { 13, 10 }
+--CRUS-HOME
+DEDLayout_l5["HOME_WIND_AT_ALTITUDE"] = { 7, 4 }
+DEDLayout_l5["HOME_WIND_AT_ALTITUDE_Value"] = { 13, 10 }
+--FIX
+DEDLayout_l5["FIX GPS ACCURACY label"] = { 4, 9 }
+DEDLayout_l5["FIX GPS ACCURACY value"] = { 14, 4 }
+--A CAL MAN
+DEDLayout_l5["ACAL_POS_DELTA"] = { 8, 5 }
+DEDLayout_l5["ACAL_POS_DELTA_label"] = { 2, 3 }
+DEDLayout_l5["DELTA_POS"] = { 16, 7, 0, "_inv", "I" }
+--A CAL AUTO
+DEDLayout_l5["DTS_ACCURACY_VPU_label"] = { 21, 2 }
+DEDLayout_l5["DTS_ACCURACY_VPU_value"] = { 23, 1 }
+--HTS MAN
+DEDLayout_l5["T4"] = { 1, 1 }
+DEDLayout_l5["T4_code"] = { 3, 4, 0, "_inv", "I" }
+DEDLayout_l5["Asterisks_T4_both"] = { 2, 1, 7, "", "I" }
+DEDLayout_l5["T8"] = { 10, 1 }
+DEDLayout_l5["T8_code"] = { 12, 4, 0, "_inv", "I" }
+DEDLayout_l5["Asterisks_T8_both"] = { 11, 1, 16, "", "I" }
+--HTS SEAD
+DEDLayout_l5["HTS_TOT_lbl"] = { 2, 3 }
+DEDLayout_l5["HTS_TOT"] = { 7, 10 }
+
+--DEDLayout_l5[""] = {,}
+
+DEDLayout = { DEDLayout_l1, DEDLayout_l2, DEDLayout_l3, DEDLayout_l4, DEDLayout_l5 }
+
+---------------------------- DED Display Utility Functions
+local function mergeString(original_string, new_data, location)
+	local new_data_length = string.len(new_data)
+	local before = string.sub(original_string, 1, location)
+	local after = string.sub(original_string, location + new_data_length + 1)
+	local base = string.sub(original_string, location + 1, location + new_data_length)
+	local merged = {}
+
+	for i = 1, new_data_length do
+		local curr_base = string.sub(base, i, i)
+		if curr_base ~= " " then
+			merged[i] = curr_base
+		else
+			merged[i] = string.sub(new_data, i, i)
+		end
+	end
+	return before .. table.concat(merged) .. after
+end
+
+function IntToByteString(intval)
+	assert(intval >= 0)
+	assert(intval <= 0xFFFFFFFF) --- (2^32 -1) ::4294967295
+	-- convert value (a float from 0.0 to 1.0) to a 16-bit signed integer from 0 to 65535
+	local retBytes = { 0, 0, 0, 0 }
+	local i = 1 -- unsigned long start from low couple of byte
+
+	while intval > 0 do
+		retBytes[i] = intval % 256
+		intval = (intval - retBytes[i]) / 256
+		i = i + 1
+	end
+	return string.char(retBytes[1], retBytes[2], retBytes[3], retBytes[4])
+end
+
+OR, XOR, AND = 1, 3, 4
```

Comment:
These should probably be local
```suggestion
local OR, XOR, AND = 1, 3, 4
```